__tests__/app.test.js

```diff
@@ -27,6 +27,36 @@ describe("app", () => {
         });
     });
   });
+  describe("/api", () => {
+    test("200: responds with a status of 200", () => {
+      return request(app).get("/api").expect(200);
+    });
```

Comment:
It could be said this is redundant since you're also testing for a 200 right after this test